Blazor_Foundation_6_Test_Shared/Documentation/Navigation/MenuDrilldown.razor

```diff
@@ -1,6 +1,152 @@
+
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="primary">
+                <p>
+                    The drilldown menu takes on the height of the tallest menu in the hierarchy, so the menu doesn't change height as
+                    the user navigates it.
+                </p>
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+
+            <OpenCodeDev.Blazor.Foundation.Components.Navigation.MenuDrilldown Vertical="true">
+                <li>
+                    <NavLink href="#">Item 1</NavLink>
+                    <MenuNested Vertical="true">
+                        <li><NavLink href="#">Item 1A</NavLink></li>
+                        <li><NavLink href="#">Item 1B</NavLink></li>
+                        <li><NavLink href="#">Item 1C</NavLink></li>
+                        <li><NavLink href="#">Item 1D</NavLink></li>
+                    </MenuNested>
+                </li>
+                <li>
+                    <NavLink href="#">Item 2</NavLink>
+                    <MenuNested Vertical="true">
+                        <li><NavLink href="#">Item 2A</NavLink></li>
+                        <li><NavLink href="#">Item 2B</NavLink></li>
+                        <li><NavLink href="#">Item 2C</NavLink></li>
+                        <li><NavLink href="#">Item 2D</NavLink></li>
+                    </MenuNested>
+                </li>
+                <li>
+                    <NavLink href="#">Item 3</NavLink>
+                    <MenuNested Vertical="true">
+                        <li><NavLink href="#">Item 3A</NavLink></li>
+                        <li><NavLink href="#">Item 3B</NavLink></li>
+                        <li><NavLink href="#">Item 3C</NavLink></li>
+                        <li><NavLink href="#">Item 3D</NavLink></li>
+                    </MenuNested>
+                </li>
+                <li><NavLink href="#">Item 4</NavLink></li>
+            </OpenCodeDev.Blazor.Foundation.Components.Navigation.MenuDrilldown>
+        </SectionLeft>
+        <hr />
+    </SectionWrapper>
+</SectionWrapper>
+
+<SectionWrapper>
```

Comment:
You don't follow Foundation 6's menu.... so we cannot see the resize like we do on Foundation 6 doc... did you test that?